client/src/App.jsx

```diff
@@ -2,16 +2,22 @@ import React from 'react';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import Login from './components/Login';
 import './App.css';
+import SearchSidebar from './components/searchSidebar/sidebar';
+import Home from './components/Homepage/index';
 import NewTicket from './components/NewTicket';
 import MainSidebar from './components/MainSidebar';
 
 export default () => (
   <BrowserRouter>
     <Switch>
+      <Route exact path="/" component={() => <div />} />
+      <Route path="/search" render={() => <SearchSidebar />} />
+      <Route path="/tickets/all/pending" component={() => <Home pendingClass="selected" />} />
```

Comment:
pending class and closed class are arbitrary names. Is there another pending class? if not it would be better to just use the keyword ```className``` as it serves the purpose.

Reply:
@Anies12 Another approach you could take, instead of passing a class at all, is to be a little more abstract and just pass a "selectedTicketType" prop (and then the value can be "pending" or "closed").  Then the Home component can use this to toggle selected classes on its elements.  In other words, Home is responsible for the visual presentation for different ticket types (instead of togging a class, maybe in the future it simply includes different child components).
Then if you add a new path (for example tickets/all/open) you don't need a new prop, you can reuse the prop that you already have instead of introducing a new prop.  Also, it prevents impossible combinations of props like `<Home pendingClass="selected" closedClass="selected" />`

If you really want to be fancy, you could go even further and use a route match param to pass this in:
`<Route path="/tickets/all/:ticketType" component={Home} />` and then inside Home you'd access it at `this.props.match.params.ticketType`.

Reply:
well, I'm not used to style react components yet, thank u

Reply:
Yep! I understand.  Your code is working so these are just ideas for the future :)